### PR TITLE
Add launch timing log

### PR DESCRIPTION
**Pull request by paulproteus**
See https://gist.github.com/paulproteus/5e0ba8863d8ed17caeee625ed148deff

This is to help ensure that we are fixing the right things, when working on app startup time.

## Manual testing

I've tested that my helloworld app based on https://toga.readthedocs.io/en/latest/tutorial/tutorial-1.html seems to still launch correctly.

### CODE REVIEW

File: {{ cookiecutter.formal_name }}/app/src/main/java/org/beeware/android/MainActivity.java [hunk 1]
```diff
@@ -108,27 +113,36 @@ private void setPythonEnvVars(String pythonHome) throws IOException, ErrnoExcept
...
         Python.run("{{cookiecutter.module_name}}", new String[0]);
+        Log.d(APP_LAUNCH_TAG, "Python.run() end");
+        Log.d(APP_LAUNCH_TAG, "startPython() end");
     }
 
     protected void onCreate(@Nullable Bundle savedInstanceState) {
+        Log.d(APP_LAUNCH_TAG, "onCreate() start");
         super.onCreate(savedInstanceState);
         this.captureStdoutStderr();
+        Log.d("MainActivity.onCreate", "captured stdout/stderr");
```
Comment (freakboy3742) [inline {{ cookiecutter.formal_name }}/app/src/main/java/org/beeware/android/MainActivity.java:145]: Is this deliberately *not* APP_LAUNCH_TAG?

File: {{ cookiecutter.formal_name }}/app/src/main/java/org/beeware/android/MainActivity.java [hunk 2]
```diff
@@ -139,17 +153,24 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             Log.e("MainActivity", "Failed to create Python app", e);
```
Comment (freakboy3742) [inline {{ cookiecutter.formal_name }}/app/src/main/java/org/beeware/android/MainActivity.java:153]: Should these historical logging references be updated to APP_LAUNCH_TAG?

Comment (paulproteus) [reply {{ cookiecutter.formal_name }}/app/src/main/java/org/beeware/android/MainActivity.java:153]: Seems smart, yeah!

File: {{ cookiecutter.formal_name }}/app/src/main/java/org/beeware/android/MainActivity.java (hunk 1 above)
Comment (paulproteus) [reply {{ cookiecutter.formal_name }}/app/src/main/java/org/beeware/android/MainActivity.java:145]: I think APP_LAUNCH_TAG would be a better approach, yeah. Good catch.

**Comment by paulproteus** [comment]
You asked:

> My biggest question is around Android convention; is a full dotted path to the classname the best tag here? It seems... verbose.

It's not :) I just like being verbose.

Let me see if I can come up with a less verbose convention!

**Comment by paulproteus** [comment]
@freakboy3742 Ready for re-review. It's simpler and cleaner. The `TAG` convention I'm **now** using is consistent with Android platform conventions; see e.g. https://developer.android.com/studio/debug/am-logcat#WriteLogs

**Pull request merged**